Source/Route.swift

```diff
@@ -16,35 +16,31 @@
 
 import UIKit
 
-public class Route {
+class Route {
     
-    static let routeParameterPattern = ":[a-zA-Z0-9-_]+"
-    static let urlParameterPattern = "([^/]+)"
+    enum Pattern: String {
+        case RouteParam = ":[a-zA-Z0-9-_]+"
+        case UrlParam = "([^/]+)"
+    }
     
-    let routeParameter: NSRegularExpression = try! NSRegularExpression(pattern: routeParameterPattern, options: .CaseInsensitive)
-    let urlParameter: NSRegularExpression = try! NSRegularExpression(pattern: urlParameterPattern, options: .CaseInsensitive)
+    let routeParameter: NSRegularExpression = try! NSRegularExpression(pattern: Pattern.RouteParam.rawValue, options: .CaseInsensitive)
+    let urlParameter: NSRegularExpression = try! NSRegularExpression(pattern: Pattern.UrlParam.rawValue, options: .CaseInsensitive)
```

Comment:
:+1: and also I kept the force unwrap there because it's basically static, but other then that we shouldn't ever need to force unwrap